model/gen-ai/spans.yaml

```diff
@@ -260,6 +270,12 @@ groups:
       - ref: gen_ai.data_source.id
         requirement_level:
           conditionally_required: if applicable.
+      - ref: gen_ai.system_instructions
+        requirement_level: opt_in
+      - ref: gen_ai.input.messages
+        requirement_level: opt_in
+      - ref: gen_ai.output.messages
+        requirement_level: opt_in
```

Comment:
I'm still worried about https://github.com/open-telemetry/semantic-conventions/pull/2179#discussion_r2262761022. Is this supposed to be here?

Reply:
Removing these lines doesn't remove those attributes because they are inherited https://github.com/open-telemetry/semantic-conventions/blob/fe7e61067e25a61e874c0844aa886f50d569c2c2/model/gen-ai/spans.yaml#L247

which has been there since this span was added https://github.com/open-telemetry/semantic-conventions/pull/1900.

Reply:
Created https://github.com/open-telemetry/semantic-conventions/issues/2632 to follow up